apps/wallet-api-tools/src/components/Message.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { CSSProperties, Suspense } from "react";
+import CodeMirror, { Extension } from "@uiw/react-codemirror";
+import { langs } from "@uiw/codemirror-extensions-langs";
+import { MessageIn, MessageOut } from "../types";
+
+type Props = {
+  message: MessageIn | MessageOut;
+  style?: CSSProperties;
+  theme?: Extension;
+  date?: string;
+};
+
+const Message = ({ message, theme, date }: Props) => {
+  return (
+    <>
+      <p className="text-slate-400 text-sm mb-2">{date}</p>
+      <Suspense fallback={"..."}>
```

Comment:
Not sure about the usage of `Suspense` here as everything is sync under that component AFAIU